meerk40t/gui/wxmmain.py

```diff
@@ -163,7 +164,17 @@ def __init__(self, *args, **kwds):
         self.main_menubar = wx.MenuBar()
         self.__set_menubars()
 
-        self.main_statusbar = self.CreateStatusBar(3)
+        self.main_statusbar = self.CreateStatusBar(4)
+        self.main_statusbar.SetStatusStyles([wx.SB_SUNKEN] * self.main_statusbar.GetFieldsCount())
+        self.main_statusbar.SetStatusWidths([-1] * self.main_statusbar.GetFieldsCount())
+        self.SetStatusBarPane(0)
+        self.main_statusbar.SetStatusText(_("Status..."), 0)
+        self.Bind(wx.EVT_MENU_OPEN, self.on_menu_open) # ToDo DELETE - only for testing
```

Comment:
Testing

Reply:
Actually - this needs to stay - it's just the comment that needs to go.